autogpt_platform/backend/backend/blocks/llm.py

```diff
@@ -35,6 +35,7 @@
     ProviderName.OLLAMA,
     ProviderName.OPENAI,
     ProviderName.OPEN_ROUTER,
+    ProviderName.AIML,
```

Comment:
```suggestion
    ProviderName.AIML_API,
```
same for all other occurrences